src/components/Header/Header.tsx

```diff
@@ -1,26 +1,47 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import styles from './Header.module.scss';
 
 const Header: React.FunctionComponent = () => {
   return (
-    <header style={{ padding: 20 }}>
-      <h1>Nice Weather App</h1>
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/cities">Cities</Link>
-          </li>
-          <li>
-            <Link to="/about">About</Link>
-          </li>
-          <li>
-            <Link to="/contacts">Contacts</Link>
-          </li>
-        </ul>
-      </nav>
+    <header className={styles.header}>
+      <div className={styles.container}>
+        <div className={styles.title}>
+          <h1>Nice Weather App</h1>
+          <div className={styles.logo}></div>
+        </div>
+        <nav className={styles.nav}>
+          <ul className={styles.linkContainer}>
+            <li className={styles.link}>
+              <NavLink to="/">Home</NavLink>
```

Comment:
just fyi: you can create config for nav links and use `.map()` to generate ui. in this case you'll be sure html is the same for every link and you didn't miss anything (for example, active class name for home link)